composables/transaction/transactionBuy.ts

```diff
@@ -9,8 +9,27 @@ import { getApiCall } from '@/utils/gallery/abstractCalls'
 import { payRoyaltyTx, somePercentFromTX } from '@/utils/support'
 import type { ActionBuy } from './types'
 import { verifyRoyalty } from './utils'
+import { existentialDeposit } from '@kodadot1/static'
+import { decodeAddress, encodeAddress } from '@polkadot/util-crypto'
+const formatAddress = (address: string): string => {
+  const { chainProperties } = useChain()
 
-function payRoyaltyAssetHub(
+  const publicKey = decodeAddress(address)
```

Comment:
, you can directly pass address into encode. 

+ we have already utils for that in kodadot

Reply:
>you can directly pass address into encode.

done

>we have already utils for that in kodadot

didn't find something that was more convenient to use then `encodeAddress` directly